packages/oui-status/status.less

```diff
@@ -0,0 +1,25 @@
+@import './_normalize';
+@import './_mixins';
+
+.oui-status {
+  @oui-status-selector: oui-status;
+
+  #oui > .status-normalize();
+  #oui > .status-base();
+
+  &_success {
+    #oui > .status-success();
```

Comment:
New rules (again again)! 🎉
Don't write mixins if it's not use elsewhere (or maybe if it's not too verbose imho).